blast/blasthit.cpp

```diff
@@ -18,10 +18,10 @@
 
 #include "blasthit.h"
 
-#include "../graph/debruijnnode.h"
```

Comment:
Can we somehow sort includes while here?

E.g.

"local ones"
"external, but local to project"
"QT"
"C++ standard"
"C standard"

Reply:
Suggest doing that in the MR fixing code style. I'll set up clang-format and format check in the pipeline.

Now I don't know how to sort includes automatically with clang-format without reformatting the whole project source )